translator/trace/opentelemetry/otlp_to_oc.go

```diff
@@ -0,0 +1,10 @@
+package opentelemetry
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector/consumer/consumerdata"
+	tracepb "github.com/open-telemetry/opentelemetry-proto/gen/go/trace/v1"
+)
+
+func ResourceSpansToTraceData(resourceSpans *tracepb.ResourceSpans) (consumerdata.TraceData, error) {
+	panic("Not implemented")
```

Comment:
TODO: implement

Reply:
We have the opposite translation already: https://github.com/open-telemetry/opentelemetry-collector/blob/master/translator/trace/oc_to_otlp.go
May be useful to consult when implementing this one.